ci/update-packages.sh

```diff
@@ -29,8 +29,8 @@ if ! [ -x "$(command -v docker)" ]; then
   exit 1
 fi
 
-# Check if we can run docker without sudo
-if ! docker ps > /dev/null 2>&1; then
+# Check if we can run docker without sudo (check is not needed for Macos system)
```

Comment:
Ah interesting. I didn't know [that](https://www.reddit.com/r/docker/comments/byciui/why_can_macos_run_docker_without_sudo_root_but_on/).

Reply:
Me neither =)